unittest/spatial.cpp

```diff
@@ -737,6 +737,83 @@ BOOST_AUTO_TEST_CASE(test_Inertia)
 
   // Test disp
   std::cout << aI << std::endl;
+
+  // Test Inertia parametrizations
+  {
+    typedef LogCholeskyParametersTpl<double, 0> LogCholeskyParameters;
+    typedef PseudoInertiaTpl<double, 0> PseudoInertia;
+
+    Inertia::Vector10 eta;
+    eta.setRandom();
+    LogCholeskyParameters log_cholesky = LogCholeskyParameters(eta);
+
+    // Convert logcholesky parametrization to pseudo-inertia
+    Eigen::Matrix4d pseudo = log_cholesky.toPseudoInertia().toMatrix();
+    // Convert logcholesky to inertia tpl
+    Inertia I_from_log_cholesky = Inertia::FromLogCholeskyParameters(log_cholesky.parameters);
+
+    // Check if conversion from inertia tpl to pseudo inertia gives same result as from logcholesky
+    // parametrization to pseudo-inertia
+    Eigen::Matrix4d pseudo_from_inertia = I_from_log_cholesky.toPseudoInertiaMatrix();
+    BOOST_CHECK(pseudo.isApprox(pseudo_from_inertia, 1e-10));
+
+    // // Check if log-cholesky parametrization to pseudo-inertia gives same result as their
+    // calculations double alpha = log_cholesky.alpha, d1 = eta[1], d2 = eta[2], d3 = eta[3]; double
+    // s12 = eta[4], s23 = eta[5], s13 = eta[6]; double t1 = eta[7], t2 = eta[8], t3 = eta[9];
+
+    double exp_alpha = std::exp(log_cholesky.alpha);
+    double exp_d1 = std::exp(log_cholesky.d1);
+    double exp_d2 = std::exp(log_cholesky.d2);
+    double exp_d3 = std::exp(log_cholesky.d3);
+
+    Eigen::Matrix4d U;
+    U << exp_d1, log_cholesky.s12, log_cholesky.s13, log_cholesky.t1, 0, exp_d2, log_cholesky.s23,
+      log_cholesky.t2, 0, 0, exp_d3, log_cholesky.t3, 0, 0, 0, 1;
```

Comment:
```suggestion
    // clang-format off
    U << exp_d1, log_cholesky.s12, log_cholesky.s13, log_cholesky.t1,
         0, exp_d2, log_cholesky.s23, log_cholesky.t2,
         0, 0, exp_d3, log_cholesky.t3,
         0, 0, 0, 1;
    // clang-format on
```

Reply:
Hi @jorisv,

Thank you for your detailed review. I appreciate your feedback and will address all the related comments over the weekend. If I encounter any difficulties, I will reach out for assistance.

Regarding the Boost.Python binding, I will do my best to implement it at the beginning of next week. Since this will be my first time working with Boost.Python, I might need some additional time and possibly your guidance. If I struggle with any aspects, I will let you know and may require your help.

Reply:
Hi @jorisv,

Thank you for your detailed review. I appreciate your feedback and will address all the related comments over the weekend. If I encounter any difficulties, I will reach out for assistance.

Regarding the Boost.Python binding, I will do my best to implement it at the beginning of next week. Since this will be my first time working with Boost.Python, I might need some additional time and possibly your guidance. If I struggle with any aspects, I will let you know and may require your help.